### ISSUE TITLE
How to extract pool id

### ISSUE THREAD

**Issue by IlluvatarEru** [IlluvatarEru/mev-inspect-py#24]

**Comment by 0xpanoramix** [comment]
There might be something quick and dirty to do with kubectl tools using sed or awk. The advantage would be to keep everything in the bash script.

**Comment by IlluvatarEru** [comment]
Yes I think that would work, it's what we're looking for. Please raise a PR when you have it
